### ISSUE TITLE
Error in Calculating Flops

### ISSUE THREAD

**Issue by Allhailankurgupta** [1adrianb/pytorch-estimate-flops#6]
## I was using this code below 
```
class LeNet(nn.Module):
    def __init__(self):
        super(LeNet, self).__init__()
        self.conv1 = nn.Conv2d(1, 6, 5, padding=2)
        self.conv2 = nn.Conv2d(6, 16, 5)
        self.fc1   = nn.Linear(16*5*5, 120)
        self.fc2   = nn.Linear(120, 84)
        self.fc3   = nn.Linear(84, 10)

    def forward(self, x):
        x = F.max_pool2d(F.relu(self.conv1(x)), (2, 2))
        x = F.max_pool2d(F.relu(self.conv2(x)), (2, 2))
        x = x.view(-1, self.num_flat_features(x))
        x = F.relu(self.fc1(x))
        x = F.relu(self.fc2(x))
        x = self.fc3(x)
        return x

    def num_flat_features(self, x):
        size = x.size()[1:]
        return np.prod(size)

new_model = LeNet()
new_model.to(DEVICE)
from pthflops import count_ops
false_inp = torch.rand(1,1,28,28).to(DEVICE)
# # Count the number of FLOPs
v = count_ops(new_model, false_inp)
```
## and I got this error
![Screenshot (41)](https://user-images.githubusercontent.com/33093973/98875919-2d1bd500-24a3-11eb-9e61-a763184f47c4.png)
**It was working perfectly fine a few days back
Could you please resolve this issue**

**Comment by 1adrianb** [comment]
Hi @Allhailankurgupta,

I tried running your code but I was unable to reproduce your error using pytorch 1.6/1.7 & pthflops 0.3.3
Could you please let me know what versions you are using?

**Comment by Allhailankurgupta** [comment]
I was running the code on colab so didn't pay that much attention to the versions
But I checked I am using 
pytorch 1.7
And pthflops 0.3.3

**Comment by Allhailankurgupta** [comment]
I am facing the same error with the code given as illustration for resnet18

**Comment by 1adrianb** [comment]
Unfortunately I am unable to reproduce it. I've also just triggered a build (https://travis-ci.com/github/1adrianb/pytorch-estimate-flops/builds/200193700) and as you can see the code run successfully.  
Perhaps there is other code around that interferes?

**Comment by Allhailankurgupta** [comment]
It is kind of strange 
I just installed pthflops on my local machine 
It gave no error at all

But the exact same code is giving error on colab

Can you please check if the error comes on colab?

**Comment by Allhailankurgupta** [comment]
I just checked it is working with
pytorch 1.6.0 but not with pytorch 1.7.0

**Comment by 1adrianb** [comment]
Fixed in 0.3.4
